### PR TITLE
some player & entity mappings

### PR DESCRIPTION
**Pull request by Kahzerx**

### CODE REVIEW

**Comment by Kahzerx** [comment]
nvm

File: mappings/17w46a#17w47a.tinydiff [hunk 1]
```diff
@@ -365,10 +365,10 @@ c	net/minecraft/unmapped/C_0447546
...
 		p	9			p_9
 c	net/minecraft/unmapped/C_0539808
-	m	(F)F	m_0053768		m_0053768
-		p	1			p_1
-	m	(F)F	m_3269987		m_3269987
-		p	1			p_1
+	m	(F)F	m_0053768		getPitch
+		p	1			delta
+	m	(F)F	m_3269987		getYaw
+		p	1			delta
```
Comment (SpaceWalkerRS) [inline mappings/17w46a#17w47a.tinydiff:371]: similar to above, should be `tickDelta`

**Review by SpaceWalkerRS — CHANGES_REQUESTED** [review]
looks good, just a few tweaks needed to upkeep our conventions

**Comment by Kahzerx** [comment]
fixed!

**Review by SpaceWalkerRS — APPROVED** [review]
ecellent, I'll wait for the workflow to run and then merge it in

**Pull request merged**